build.sbt

```diff
@@ -113,7 +108,7 @@ lazy val extras = project
   .settings(sonatypeSettings: _*)
   .settings(
     name := "sbt-jmh-extras",
+    scalaVersion := "2.12.2",
```

Comment:
Yeah, extras is java only so the question is more about what jdk target should be set here.
We're also fine in akka with 8+ nowadays I think